resources/views/layouts/default/profile/index.blade.php

```diff
@@ -133,11 +133,11 @@
         <div class="text-right">
           @if (isset($acars) && $acars === true)
           <a href="{{ route('frontend.profile.acars') }}" class="btn btn-primary"
-             onclick="alert('Save to \'My Documents/phpVMS\'')">ACARS Config</a>
+             onclick="alert('Copy or Save to \'My Documents/phpVMS\'')">ACARS Config</a>
           &nbsp;
           @endif
           <a href="{{ route('frontend.profile.regen_apikey') }}" class="btn btn-warning"
-             onclick="return confirm({{ __('Are you sure? This will reset your API key.') }})">@lang('profile.newapikey')</a>
+             onclick="return confirm('Are you sure ? This will reset your API key !')">@lang('profile.newapikey')</a>
```

Comment:
Remove the spaces from before the `?` and `!`. Otherwise, looks good

Reply:
Fixed :)